server/templates/donate-form.html

```diff
@@ -35,16 +31,22 @@ <h1 class="splash__headline">
   </section>
 
   <main class="main">
+    {% if use_thermometer %}
+    {% include "includes/thermometer.html" %}
+  {% endif %}
```

Comment:
**Nitpick (non-blocking):** I'm being so annoying but can you fix the indentation here

Reply:
Also fixed in [5fd0d92](https://github.com/texastribune/donations/pull/1231/commits/5fd0d927f22755b9e70d24cd5916b0b7ae2ff404)